primary/src/block_synchronizer/handler.rs

```diff
@@ -247,4 +258,46 @@ impl<PublicKey: VerifyingKey> Handler<PublicKey> for BlockSynchronizerHandler<Pu
 
         results
     }
+
+    #[instrument(level = "debug", skip_all)]
```

Comment:
Do we want to emit an event in case of `err`?

Reply:
Bump

Reply:
I've added `error` events inside the method